client/src/main/java/io/pravega/client/segment/impl/SegmentInputStream.java

```diff
@@ -94,4 +94,13 @@ default void setOffset( long offset) {
      * @return 0 if data read is blocking.
      */
     public int bytesInBuffer();
+
+    /**
+     * A boolean indicating if all events in the segment read completely.
+     * This is true when the segment is sealed and all events in the segment have already been read or segment have been
+     * read till end offset.
+     *
```

Comment:
Will this return true in case if segment is not sealed and reached to end offset ?

Reply:
No it will return if the segment is sealed. Updated the java doc.